packages/block-library/src/heading/index.js

```diff
@@ -30,15 +31,21 @@ export const settings = {
 		const { content, level } = attributes;
 
 		const customName = attributes?.metadata?.name;
+		const contentHTML =
+			typeof content === 'string'
```

Comment:
Can `content` ever be falsy (i.e. `undefined`). Within `toHTMLString` it winds up calling `toTree` which does some destructuring on `value` here: https://github.com/WordPress/gutenberg/blob/d1bbd9f9badb5a24866621ec9b4b0afb83bbb068/packages/rich-text/src/to-tree.js#L146

So I was just wondering if it'd be worth guarding the left-hand part of this ternary for that case with something like `typeof content === 'string' || ! content` so that `toHTMLString` is only called if it's truthy?

Apologies if I'm overthinking this!